### ISSUE TITLE
Healenium

### ISSUE THREAD

**Issue by onlybiju** [serenity-bdd/serenity-core#3248]
Did anyone try to integrate Healenium with Serenity?
Hi @wakaleo - Do you know its possible?
https://healenium.io/

Thank you!

**Comment by wakaleo** [comment]
I'm not sure if Healanium works with the latest version of Selenium. I've done a simple integration in this branch, but the Healanium setup code fails: https://github.com/serenity-bdd/serenity-core/tree/healenium-integration

**Comment by onlybiju** [comment]
We will give a try..thank you

**Comment by baskarcsb** [comment]
any luck?

**Comment by wakaleo** [comment]
Does Healenium work with the latest version of Selenium? The docs are very sparse, and the code samples (without using Serenity, just with plain Selenium and Chrome) do not appear to work.

**Comment by onlybiju** [comment]
> any luck?

No, Healenium is using older version of Selenium. I reached out to the support team. they replied below
"Thank you for highlighting the issue. The team has plans to build integrations with various frameworks, including Serenity."
But they did not give any timelines
